environment_unix.yml

```diff
@@ -2,16 +2,18 @@ name: lp
 channels:
   - conda-forge
 dependencies:
-  - bison=3.4
-  - cmake
-  - flake8
-  - llvmdev=11.0.1
-  - make
-  - numpy
+  - git
   - pip
-  - python=3.10.2
+  - make
   - re2c
-  - setuptools
   - toml
   - zlib
-  - git
+  - flex
```

Comment:
Why is flex needed? I don't think it's needed.

Reply:
Oh. I am sorry for the mistake. I recall installing `flex` when I ran into an error with `bison`. I recalled [this](https://stackoverflow.com/questions/7320370/installation-of-flex-and-bison-in-ubuntu) thread.

I am making the amends.